recipes/setuptools-github/LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
```

Comment:
This should be taken from the tarball. Can you remove this file?

Reply:
done